python/dgl/graphbolt/base.py

```diff
@@ -56,6 +57,49 @@ def isin(elements, test_elements):
     return torch.ops.graphbolt.isin(elements, test_elements)
 
 
+def csc_to_coo(indptr, node_ids=None, dtype=None, output_size=None):
```

Comment:
according to the name, I thought it converts csc(indptr, indices) to coo(src, dst). why not in this case?

Reply:
src is simply equal to indices so it is left out from the parameters and the return value. This function is mainly to convert the indptr (dst) part of the csc to coo conversion. Do you have a better naming suggestion?

Reply:
then this function is more about internal utility instead of a common one. Or in what kind of scenarios user needs to call this api.

Reply:
This function is very efficient at what it does as the PR description shows. If there are advanced users who would like to perform this conversion, they might utilize it. We might also utilize it to convert to PyG coo format without converting into a DGLBlock first. Anyone looking for a more efficient `torch.repeat_interleave` would use it as well.

However, if you think that we can leave it out from the documentation website for now, I can delete the entry from the `.rst` file.

Reply:
Let's rename it to make it easier to understand, I was also confused for several minutes until I saw this discussion.

- either put indices as parameter and return as src in the return parameter (this is no op anyway)
- or change the name to csc_indptr_to_coo_dst or similar.

For APIs, it is very important to avoid confusion, which is error prone for both users and ourselves.

Reply:
Also, make sure we have a benchmark test case to cover this code.

Reply:
@caojy1998 let's add a benchmark for this API when this PR is merged.

Reply:
OK.

Reply:
@frozenbugs, do you prefer `csc_indptr_to_coo_dst` or `indptr_to_coo`?

Reply:
Actually, what about expand_indptr?

Reply:
I wouldn't mind. Sounds good to me.

Reply:
If we find a better name, we can modify it before the release :).

Reply:
Changed all occurrences of the API to `expand_indptr`.